en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-configuration.md

```diff
@@ -0,0 +1,111 @@
+# Amazon Lambda Configuration
```

Comment:
Can we change the title of this and file name to something else. Right now we refer to this kind of content as configuration: https://github.com/wso2/docs-ei/blob/ConnectorTest/en/micro-integrator/docs/references/connectors/file-connector/file-connector-config.md

Reply:
Title changed as "Amazon Web Services Configuration".